File: dojo_11/java/src/main/java/steps/Review.java
package steps;

public class Review implements CheckoutStep{

	private Envio envio;
	private Rapipago medioDePago;

	@Deprecated
	public Review() {}


	public Review(Envio envio, Rapipago medioDePago) {
		this.envio = envio;
		this.medioDePago = medioDePago;
	}

	public SeleccionDeEnvio modificarEnvio() {
        return new SeleccionDeEnvio(this);
    }

    public SeleccionDeMedioDePago modificarMedioDePago() {
        return new SeleccionDeMedioDePago(this);
    }
}

File: dojo_11/javascript/src/pasos/Review.js
import SeleccionDeEnvio from './SeleccionDeEnvio';
import PasoDeCheckout from './PasoDeCheckout';

class Review extends PasoDeCheckout {
    constructor() {
        super();
    }

    modificarEnvio() {
        return new SeleccionDeEnvio(true);
    }
}

export default Review;